apps/ui-kit/src/lib/components/atoms/info-box/InfoBox.tsx

```diff
@@ -17,7 +17,7 @@ export interface InfoBoxProps {
     /**
      * The supporting text of the info box (optional).
      */
-    supportingText?: string;
+    supportingText?: string | React.ReactNode;
```

Comment:
React.ReactNode already contains string
```suggestion
    supportingText?: React.ReactNode;
```